gradio/route_utils.py

```diff
@@ -357,7 +357,7 @@ def __init__(
         filename: str | None = None,
         headers: Headers | None = None,
     ) -> None:
-        super().__init__(file, size=size, filename=filename, headers=headers)
+        super().__init__(file, size=size or 0, filename=filename, headers=headers)
```

Comment:
Is this correct? Meaning if a user sets size to be `None`, then it would be set here to 0?

Reply:
This isn't a public API - I did this to try and make the type hints work but it's not needed anyways so will revert